src/services/completions.ts

```diff
@@ -2059,16 +2059,17 @@ namespace ts.Completions {
             const kind = stringToToken(entry.name);
             switch (keywordFilter) {
                 case KeywordCompletionFilters.None:
-                    // "undefined" is a global variable, so don't need a keyword completion for it.
-                    return kind !== SyntaxKind.UndefinedKeyword;
+                    // TODO: GH#23631 Includes type keywords because `Array<numb/**/` is treaded as a value location currently.

```

Comment:
isPossiblyTypeArgumentPosition we have this function to check for type argument location

Reply:
Thanks, added a note to #23631